repos/system_upgrade/el7toel8/actors/checkbrltty/libraries/library.py

```diff
@@ -0,0 +1,21 @@
+import os
+import re
+
+BrlttyConf = '/etc/brltty.conf'
+
+def check_for_unsupported_cfg():
+    migrate_file = ''
+    migrate_bt = False
+    migrate_espeak = False
+    regex_bt = re.compile(r'\b((bth)|(bluez))([:\-][0-9a-fA-F]{2}){6}\b')
+    regex_espeak = re.compile(r'^\s*speech-driver\s+es\b')
+    if os.path.exists(BrlttyConf):
+        with open(BrlttyConf) as file_check:
+            for line in file_check:
+                if not migrate_bt and regex_bt.search(line):
```

Comment:
Sorry if I oversimplified the idea and overlooked something because of that.
Can functionality in L14-L21 be achieved by the following?
```
for line in file_check:
    if regex_bt.search(line):
        migrate_bt = True
    if regex_espeak.search(line):
        migrate_espeak = True
....
migrate_file = Br1ttyConf if migrate_espeak or migrate_bt else ''
return (migrate_file, migrate_bt, migrate_espeak)
```

Reply:
nit: wondering if breaking from the loop after all 2 boolean flags are set is worth the trouble.

Reply:
```
if not migrate_bt and regex_bt.search(line):
```
It was for slight optimization, i.e. not to run the regex if not needed, but np to simplify it.

Regarding breaking the loop, it depends, I can add it.

Fixed.